src/components/app/App.tsx

```diff
@@ -22,6 +23,8 @@ const App: React.FC = () => {
             <Suspense fallback={<div>Loading...</div>}>
               <Switch>
                 <Route exact path="/" component={Home} />
+                <Route exact path="/experiment" component={Experiment} />
```

Comment:
exact is not necessary here since we don't have multiple paths starting with `/experiment`